README.md

```diff
@@ -1,4 +1,7 @@
-# HTML5Sortable
+![](https://repository-images.githubusercontent.com/275870366/15880680-ba44-11ea-888b-8211b9431f94)
+
+<h1 align="center"> HTML5Sortable </h1>
```

Comment:
Can you please set this back to "left" (e.g. remove "center") it is easier from a readability point of view, because the eye doesn't have to wander inside to fine the headline so much.

Reply:
:+1: